### ISSUE TITLE
FUCK THE ISSUES - YOU ARE THE BEST GUY AROUND HERE

### ISSUE THREAD

**Issue by cataseven** [fjramirez1987/PTZ-YCC365#16]
With this project you made my day man! I have opened this topic to thank you. You are great! What a project! Thank you

**Comment by eerison** [comment]
Yep it helped me to understand how the communication works as well,  maybe I fork the project
